core/src/main/kotlin/com/justai/jaicf/model/activation/selection/ActivationSelector.kt

```diff
@@ -0,0 +1,38 @@
+package com.justai.jaicf.model.activation.selection
```

Comment:
Are you sure this interface and its implementations should be placed to `model`?